db/migrate/20160726090624_add_slug_to_wiki_pages.rb

```diff
@@ -0,0 +1,61 @@
+class AddSlugToWikiPages < ActiveRecord::Migration
+  def up
+    add_column :wiki_pages, :slug, :string
+    add_index :wiki_pages, [:wiki_id, :slug], name: 'wiki_pages_wiki_id_slug', unique: true
+
+    migrate_titles
+    change_column_null :wiki_pages, :slug, true
```

Comment:
I've added a comment on why it's necessary. In line 3, wiki_pages still contain NULL values for the slug, and this would raise an error when trying to add a `null: false` column.